dbt/include/athena/macros/adapters/schema.sql

```diff
@@ -0,0 +1,12 @@
+{% macro default__create_schema(relation) -%}
+  {%- call statement('create_schema') -%}
+    create schema if not exists {{ relation.without_identifier().render_hive() }}
+  {% endcall %}
+{% endmacro %}
+
+
+{% macro default__drop_schema(relation) -%}
```

Comment:
nit: same as here, https://github.com/dbt-athena/dbt-athena/pull/152/files#r1112948055 maybe we prefix with `athena__`

Reply:
Yes indeed I did not check all the code in the previous commits :)